docs/proposals/parameter-distribution.md

```diff
@@ -0,0 +1,85 @@
+# Proposal for Supporting various parameter distributions in Katib
+
+## Summary
+The goal of this project is to enhance the existing Katib Experiment APIs to support various parameter distributions such as uniform, log-uniform, and qlog-uniform. Then extend the suggestion services to be able to configure distributions for search space using libraries provided in each framework.
+
+## Motivation
+Currently, Katib is limited to supporting only uniform distribution for integer, float, and categorical hyperparameters. By introducing additional distributions, Katib will become more flexible and powerful in conducting hyperparameter optimization tasks.
+
+## Proposal
+
+### Maintaining two versions of CRD APIs
```

Comment:
Could you reconstruct the API proposal based on my comment? https://github.com/kubeflow/katib/pull/2334#issuecomment-2149647637

Basically, we can not remove the existing API. So we can just add new API fields.
Also, could you mention how to keep consistently existing `parameterType` APIs?

Reply:
Please review latest changes.